pkg/schedule/config/type.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2024 TiKV Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+type CheckerSchedulerType string
```

Comment:
Why not put it under the checker package?

Reply:
Because the scheduler will also be placed here later.

Reply:
Then, how about just using type as the package name?

Reply:
sgtm